certified-connectors/Evocom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2416 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Evocom",
+    "description": "Evocom is an easy-to-use solution for collaborative workflows and task management for enterprises. According to the motto \"first digitize, then automate\", companies transform manual processes into digital processes in order to further integrate and automate them afterwards.",
+    "contact": {
+      "name": "Evocom Support",
+      "url": "https://evocom.de/kontakt",
+      "email": "support@evocom.de"
+    }
+  },
+  "host": "productivity.evocom.net",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/hooks/Tasks/New": {
+      "x-ms-notification-content": {
+        "schema": {
+          "$ref": "#/definitions/TaskResponse"
+        }
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Created"
+          }
```

Comment:
@denis-sisic Can you please add all the expected status codes in all the responses section of each operation, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.